test/acceptance/perf/setup/service-group/main.tf

```diff
@@ -0,0 +1,132 @@
+resource "aws_ecs_service" "test_server" {
+  name            = "${var.name}-test-server"
+  cluster         = var.cluster_arn
+  task_definition = module.test_server.task_definition_arn
+  desired_count   = var.server_instances_per_service_group
+  network_configuration {
+    subnets = var.private_subnets
+  }
+  propagate_tags         = "TASK_DEFINITION"
+  enable_execute_command = true
+
+  tags = var.tags
+  capacity_provider_strategy {
+    capacity_provider = "FARGATE_SPOT"
```

Comment:
One concern with spot capacity is it can be pulled out from under us (interrupting our tasks / etc). 

Spot capacity seems useful to minimize cost while dev/testing the performance test itself, but when running the test proper, we should probably not use spot capacity to ensure consistency from test run to test run?

Reply:
I'm going to try out spot instances to see if we can save some money. If they start causing problems, we can go back to regular instances.